src/components/language/language.js

```diff
@@ -1,68 +1,41 @@
 import React, { useState, useEffect } from "react";
+import Radio from "../language/radiobuttons";
 import "./language.css";
 
 const Language = props => {
   const [language, setLanguage] = useState();
   const { game } = props;
   useEffect(() => {
     setLanguage(game.language);
-  });
+  }, [game]);
 
   const onChange = e => {
     setLanguage(e.target.value);
     game.setlanguage(e.target.value);
   };
-
+  const languageObj = {
+    English: "en",
+    French: "fr",
+    Kinyarwanda: "krw"
+  };
+  let gameLanguages = [];
+  for (const languageKey in languageObj) {
```

Comment:
I would replace for loop with map here, and change the language object to an array so that it is easier to iterate over. Personal opinion!

```js
const gameLanguages = [
  ['English', 'en'],
  ['French', 'fr'],
  ['Kinyarwanda', 'krw'],
].map(([long, short]) => (
  return <Radio...
));
```